index.js

```diff
@@ -23,10 +23,9 @@ app.post('/generate', (req, res) => {
       .insert(qrCode)
       .into('qrcodes')
 
-    query.then(() => console.log('done!'))
+    query
+      .then(() => res.json(qrCode))
```

Comment:
you should return here in the response the object that knex gives you. To do so, chain a `.returning('*')` after the `.into()`